accounts/views.py

```diff
@@ -37,7 +30,7 @@ def create(self, request, *args, **kwargs):
         return Response({'user': out}, status=status.HTTP_201_CREATED)
 
 
-class UserProfileAPIView(generics.RetrieveUpdateDestroyAPIView):
+class UserProfileAPIView(generics.RetrieveAPIView, generics.UpdateAPIView):
```

Comment:
Why need use two classes?
I sent you link https://www.django-rest-framework.org/api-guide/generic-views/#retrieveupdateapiview yesterday :)
```suggestion
class UserProfileAPIView(generics.RetrieveUpdateAPIView):
```